exercicios-para-casa/mariana-seidel/uriJudge/04-exercicio/script.js

```diff
@@ -0,0 +1,28 @@
+let codigo
```

Comment:
let codigo
let alcool=0
let gasolina=0
let diesel=0

while (codigo!== 4){
    codigo = Number(prompt('Insira aqui o código do combustível'))

    switch (codigo){
        case 1:
            alcool++
            break;
        case 2:
            gasolina++
            break;
        case 3:
            diesel++
            break;
    }
    if(codigo === 4) {
      alert(`Muito obrigada!\nÁlcool: ${alcool}\nGasolina: ${gasolina}\nDiesel: ${diesel}`) 
      break;
    }
}

Fiz alguns ajustes no seu exercício, podemos utilizar somente um if para verificar se o codigo é igual a 4, se ele for, damos um break e o while é finalizado.